### ISSUE TITLE
Visual feature requests!

### ISSUE THREAD

**Issue by thexavier666** [anufrievroman/calcure#10]
1. Ability to "move" to a certain day manually. Main day will have the current highlight color. Hovered day will have a different color.
3. Draw a border around each day. Allows for better visual separation.
4. Draw border/underline/bold the month/year.

**Comment by anufrievroman** [comment]
Hello, thank you for the suggestions. Let me address one by one:
1. On one hand, I agree it would be cool, on the other hand, it sort of defeats the idea to add events in the fewest possible key presses. Another problem is that it would require to completely rework all the controls and remake how the user interacts with the program. So it's a major remake of the program, but I'm not convinced it is really a better UX, albeit a more intuitive one.
2. I'd say we are already quite limited in the space available for each day, assuming a 15" monitor with reasonable font size, especially in the horizontal lines. However, I'll see if it is easy to implement, then I may add it as an option.
3. Indeed, bold or underline might be an option. Not sure how to best implement it, because adding an option in the config for each UI element is a bit too much. Perhaps only options for the headers and days would be enough.

**Comment by anufrievroman** [comment]
Just added the possibility to make some UI elements bold and underlined. It will become available in version 1.8 soon.

Relevant new config options:
```
bold_today = Yes
bold_days = Yes
bold_day_names = Yes
bold_weekends = Yes
bold_weekend_names = Yes
bold_title = Yes
underlined_today = No
underlined_days = No
underlined_day_names = No
underlined_weekends = No
underlined_weekend_names = No
underlined_title = No
```

**Comment by thexavier666** [comment]
Hi, sorry for the delayed response.

Regarding point 1, basically what I wanted was something similar to VIM's `visual` mode. But I can understand your point.

Point 2, I agree with regards to why you need so much gaps. I actually use the calendar for viewing purpose. Hence, there is a lot of gap between the dates while the actual numbers look tiny in the terminal. When compared to the output of `cal`, the view looks very _"airy"_ 😄

Point 3, thanks for the new options.

**Comment by anufrievroman** [comment]
By the way, if you are looking for better visual separation of days, you can try these options:
```
minimal_today_indicator = No
minimal_days_indicator = No
minimal_weekend_indicator = No
```

**Comment by anufrievroman** [comment]
Commit 2cea14970a3be09209d1fae5fbd9b340be378a86 implements the requested boarder in the monthly view. It will be available in the upcoming version 2.0 sometime in May.

**Comment by anufrievroman** [comment]
So with issues 2 and 3 implemented and 1 not planned, I'll close this issue for now.
Feel free to comment if there are some further ideas.
